### PR TITLE
Update pet form templates

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Just some general updates to fields and labels.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1920** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1920" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1920?utm_source=stack-comment).

File: apps/store/src/services/PriceCalculator/data/SE_PET_CAT.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const SE_PET_CAT: Template = {
           field: {
             type: 'text',
             name: 'name',
-            label: { key: tKey('FIELD_PET_NAME_LABEL') },
+            label: { key: tKey('FIELD_NAME_PET_LABEL') },
```
Comment (alebedev) [inline apps/store/src/services/PriceCalculator/data/SE_PET_CAT.ts:21]: I think this key became more confusing.  `FIELD_PET_NAME_LABEL` is easier to understand

File: apps/store/src/services/PriceCalculator/data/SE_PET_DOG.ts [hunk 2]
```diff
@@ -50,15 +50,15 @@ export const SE_PET_DOG: Template = {
           field: {
             type: 'date',
             name: 'birthDate',
-            label: { key: tKey('FIELD_PET_DATE_OF_BIRTH_LABEL') },
+            label: { key: tKey('FIELD_BIRTH_DATE_PET_LABEL') },
```
Comment (alebedev) [inline apps/store/src/services/PriceCalculator/data/SE_PET_DOG.ts:53]: Same, FIELD_PET_* vs FIELD_*_PET

**Review by alebedev — APPROVED** [review]
Looks good, some localization questions

File: apps/store/src/services/PriceCalculator/data/SE_PET_DOG.ts (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/services/PriceCalculator/data/SE_PET_DOG.ts:53]: @alebedev aha yeah I was trying to namespace it the same as other things like we put DOG/CAT as suffix but we can go back to more readable keys

**Pull request merged**